src/actions/multiple_target_action.py

```diff
@@ -35,6 +34,8 @@ def cursorless_multiple_targets(m) -> str:
 
     try:
         target_separater = m.cursorless_target_separator
+        if target_separater != "at":
+            target_list[-1]["position"] = target_separater
```

Comment:
We'll need to fix this one; won't work for list targets

Reply:
Ah damn I hate weakly typed languages. Probably should make a utility for updating single or list of targets.

Reply:
@pokey I have pushed a change which updates lists but I have no idea how I'm going to add positions to a range target. Whatever I try the result is incorrect.

Reply:
Don't worry bout range targets.  Positional modifiers don't really make sense for them so do whatever you want

Reply:
Ok then Im done with this change. Pushed and ready.